File: src/com/izhan/convertanumbertoalengthstring/String.java
package com.izhan.convertanumbertoalengthstring;

import java.io.*;
import java.util.*;

public class String {
    public static java.lang.String fromLength(Length length) {
        return length.getValue() + " " + length.getUnit();
    }

    public static java.lang.String fromNumber(Number number) {
        return java.lang.String.valueOf(number.getValue());
    }
}

File: src/com/izhan/convertanumbertoalengthstring/android/String.kt
package com.izhan.convertanumbertoalengthstring.android

import java.io.*
import java.util.*

object String {
    fun fromLength(length: Length): kotlin.String {
        return "${length.value} ${length.unit}"
    }

    fun fromNumber(number: Number): kotlin.String {
        return number.value.toString()
    }
}
